api/environments/identities/constants.py

```diff
@@ -6,5 +6,5 @@
 
 identifier_regex_validator = RegexValidator(
     regex=RE_VALID_IDENTIFIER,
-    message="Identifier can only contain unicode letters, numbers, and the symbols: ! # $ %% & * + / = ? ^ _ ` { } | ~ @ . -",
+    message="Identifier can only contain unicode letters, numbers, and the symbols: ! # $ % & * + / = ? ^ _ ` { } | ~ @ . -",
```

Comment:
The double `%` here is intentional, as to prevent Django from doing string interpolation. Check out this thread: https://github.com/Flagsmith/flagsmith/pull/6024#discussion_r2337595607